Bottom: 4aaf9f8114a361c30ea25756a6932bf3c329ca7d
Top:    fbdaa0c7e0296b3b6a9a770192845413010b07b5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:54:38 +0100

Refresh of off-to-find-burried-treasure-o

---

diff --git a/src/find.c b/src/find.c
index e4c114d..af9103c 100644
--- a/src/find.c
+++ b/src/find.c
@@ -4028,17 +4028,17 @@ IsPolygonAnIsland (LayerType *layer, PolygonType *polygon)
    */
   TheFlag = FOUNDFLAG | DRCFLAG;
 
-  ResetConnections (False);
+  ResetConnections (false);
 
   /* Let the search stop if we find something we haven't yet seen */
-  drc = True;
-  User = False;
+  drc = true;
+  User = false;
 
   ListStart (POLYGON_TYPE, layer, polygon, polygon);
 
   do
     {
-      any_more = DoIt (False, False);
+      any_more = DoIt (false, false);
 
       /* Check if we got any useful hits */
       connected_count = 0;
@@ -4057,8 +4057,8 @@ IsPolygonAnIsland (LayerType *layer, PolygonType *polygon)
     }
   while (any_more);
 
-  drc = False;
-  ResetConnections (False);
+  drc = false;
+  ResetConnections (false);
   FreeConnectionLookupMemory ();
 
   return (connected_count == 0);
diff --git a/src/pour.c b/src/pour.c
index b957308..d94ec87 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -1342,7 +1342,7 @@ RestoreToPours (DataType * Data, int type, void *ptr1, void *ptr2)
 //      printf ("Calling InitPourClip from RestoreToPour\n");
       InitPourClip (PCB->Data, (LayerTypePtr) ptr1, (PourTypePtr) ptr2);
     }
-  PlowPours (Data, type, ptr1, ptr2, add_plow, False);
+  PlowPours (Data, type, ptr1, ptr2, add_plow, false);
 }
 
 void
@@ -1356,7 +1356,7 @@ ClearFromPours (DataType * Data, int type, void *ptr1, void *ptr2)
 //      printf ("Calling InitPourClip from ClearFromPour\n");
       InitPourClip (PCB->Data, (LayerTypePtr) ptr1, (PourTypePtr) ptr2);
     }
-  PlowPours (Data, type, ptr1, ptr2, subtract_plow, False);
+  PlowPours (Data, type, ptr1, ptr2, subtract_plow, false);
 }
 
 #warning FIXME Later: We could perhaps reduce un-necessary computation by using this function
@@ -1365,5 +1365,5 @@ MarkPourIslands (DataType * Data, int type, void *ptr1, void *ptr2)
 {
   if (!Data->ClipPours)
     return;
-  PlowPours (Data, type, ptr1, ptr2, mark_islands, True);
+  PlowPours (Data, type, ptr1, ptr2, mark_islands, true);
 }
